static/src/javascripts/projects/common/modules/identity/ad-prefs/wordings.js

```diff
@@ -7,30 +7,38 @@ type CheckboxWording = {
     text?: string,
 };
 
+type QuestionWording = {
+    title: string,
+    text?: string,
+};
+
 type ConsentWording = {
-    question: string,
+    question: QuestionWording,
     yesCheckbox: CheckboxWording,
     noCheckbox: CheckboxWording,
 };
 
 const ThirdPartyConsentWording: ConsentWording = {
-    question:
-        'We use cookies to improve your experience on our site and to show you relevant advertising.',
+    question: {
+        title: 'Set your personalised ads preferences',
+        text:
+            '<p>We work with <a class="u-underline" href="https://www.theguardian.com/info/cookies">advertising partners</a> (including Google and the Ozone project) to show you ads for things you might beinterested in. You can manage whether the ads you see on our sites are personalised by selecting one of the options below.</p><p>Don\'t worry, you can always revisit these settings by following the link on our <a class="u-underline" href="https://www.theguardian.com/info/cookies">cookies policy</a> page.</p>',
```

Comment:
Can we store this as react JSX to get away with not using `dangerouslySetinnerHTML` later on?